testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/authz/UmaGrantTypeTest.java

```diff
@@ -371,6 +372,37 @@ public void testObtainRptUsingAccessToken() throws Exception {
         assertTrue(permissions.isEmpty());
     }
 
+    @Test
+    public void testObtainDecisionUsingAccessToken() throws Exception {
+        AccessTokenResponse accessTokenResponse = getAuthzClient().obtainAccessToken("marta", "password");
```

Comment:
The following test pattern is missing:
[1]
- machingUri: false
- result: false
[2]
- `uri` is empty
[3]
- `uri` is empty
- `scopes` exist

Reply:
I added these tests.